README.md

```diff
@@ -102,8 +102,42 @@ fn fetch_an_integer() -> String {
 }
 ```
 
+## JSON Support
+
+Support for the RedisJSON Module can be enabled by specifying "json" as a feature in your Cargo.toml.
+
+`redis = { version = "0.17.0", features = ["json"] }`
+
+Then you can simply include the `JSONCommands` which will add the `json` commands to all Redis Connections (not to be confused with just `Commands` which only adds the default commands)
+
+```rust
+use redis::Client;
+use redis::JSONCommands;
+use redis::RedisResult;
+
+// Result returns Ok(true) if the value was set
+// Result returns Err(e) if there was an error with the server itself 
+fn set_json_bool<P: ToString>(key:P, path:P, b: bool) -> RedisResult<bool> {
+    let client = Client::open("redis://127.0.0.1").unwrap();
+    let connection = client.get_connection().unwrap();
+
+    // runs `JSON.SET {key} {path} {b}`
+    connection.json_set(key, path, b)?
+    
+    // you'll need to use serde_json (or some other json lib) to deserialise the results from the bytes
+    // It will always be a Vec, if no results were found at the path it'll be an empty array
+}
+
+```
+
 ## Development
 
+If you want to test with modules, you must:
+- Have those modules
+- Set the `REDISRS_{module-name}_PATH` environment variable
```

Comment:
I think we should add some underscores to these environment variable names to make them less surprising, `REDIS_RS_{module-name}_PATH` and `REDIS_RS_REDIS_JSON_PATH`.

Reply:
the GitHub repo for RedisJSON is [RedisJSON/RedisJSON](https://github.com/RedisJSON/RedisJSON) as opposed to Redis-Json or redis_json which gives the idea that it meant to be read as one word

Might I suggest something like `REDIS_RS_REDISJSON_PATH` which also follows the naming of RediSearch being one word, so it'd be `REDIS_RS_REDISEARCH_PATH`.

Reply:
The way I understand the SCREAMING_CASE and snake_case conventions is that internal capitals (except in acronyms) will introduce a new word and thus require a `_` separator.

I recently had a discussion with a coworker about this and asked Twitter what people thought, see also https://twitter.com/djco/status/1513935084281290763. So I would still consider `REDIS_JSON` more intuitive for an environment variable name if the project is called RedisJSON.

Reply:
You make a very good point